spec-main/api-tray-spec.ts

```diff
@@ -26,18 +28,16 @@ describe('tray module', () => {
 
   describe('tray.destroy()', () => {
     it('destroys a tray', () => {
-      expect(tray.isDestroyed()).to.be.false()
+      expect(tray.isDestroyed()).to.be.false('tray is destroyed')
```

Comment:
These strings should indicate the _expected_ case IMO not what actually happened.  i.e. following the _should_ syntax

```suggestion
      expect(tray.isDestroyed()).to.be.false('tray should not be destroyed')
```

This is because the error that gets thrown looks like this

```
[AssertionError: message here: expected false to be true]
```

So this would be

```
[AssertionError: tray should not be destroyed: expected false to be true]
```